Frameworks/CoreGraphics/CGContext.mm

```diff
@@ -1918,66 +1937,134 @@ void CGContextSetPatternPhase(CGContextRef context, CGSize phase) {
     textTransform.ty = std::round(textTransform.ty);
 
     CGAffineTransform finalTextTransform = CGAffineTransformConcat(textTransform, deviceTransform);
-    if ((fabs(finalTextTransform.a * glyphRun->fontEmSize) <= c_glyphThreshold &&
-         fabs(finalTextTransform.b * glyphRun->fontEmSize) <= c_glyphThreshold) ||
-        (fabs(finalTextTransform.d * glyphRun->fontEmSize) <= c_glyphThreshold &&
-         fabs(finalTextTransform.c * glyphRun->fontEmSize) <= c_glyphThreshold)) {
-        TraceWarning(TAG, L"Glyphs too small to be rendered");
+    for (size_t i = 0; i < runsCount; ++i) {
```

Comment:
runs within a line don't necessarily have the same font size, right?

---
In reply to: [100872254](https://github.com/Microsoft/WinObjC/pull/1933#discussion_r100872254) [](ancestors = 100872254)